From 0e45ee0b0c6b165d0ceb60a30e2f3a83e9273c6f Mon Sep 17 00:00:00 2001
From: Reddog <reddog@mastersword.de>
Date: Wed, 16 Apr 2008 20:51:25 +0100
Subject: [PATCH] gta01-fix-resume-redo-par.patch

 --> (http://bugzilla.openmoko.org/cgi-bin/bugzilla/attachment.cgi?id=595&action=view)
fix resume of fbdev  BZ#79

I played a bit with the fbdev resume. With this patch the neo doesn't get this
white screen on resume from console and from X.

Signed-off-by: Reddog <reddog@mastersword.de>
---
 drivers/video/s3c2410fb.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/video/s3c2410fb.c b/drivers/video/s3c2410fb.c
index b3c31d9..1905135 100644
--- a/drivers/video/s3c2410fb.c
+++ b/drivers/video/s3c2410fb.c
@@ -1028,6 +1028,8 @@ static int s3c2410fb_resume(struct platform_device *dev)
 
 	s3c2410fb_init_registers(fbinfo);
 
+	s3c2410fb_set_par(fbinfo);
+
 	return 0;
 }
 
-- 
1.5.6.5

